src/preprocess/base.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+
+from ..utils import Region, get_kenya
+
+
+__all__ = ['BasePreProcessor', 'Region', 'get_kenya']
+
+
+class BasePreProcessor:
+    """Base for all pre-processor classes
```

Comment:
Can we better document what the processing classes are supposed to be doing here?

e.g `Processing classes are responsible for normalizing dimensions, optionally subsetting Kenya, etc..`

Reply:
done!! thanks mate :)